cli/dts/lib.deno.unstable.d.ts

```diff
@@ -540,6 +540,177 @@ declare namespace Deno {
+     * the file in the Deno cache.
+     *
+     * By default, this is not included in the output. Set the `paths` option
+     * to `true` in the options to have this information populated. */
+    local?: string;
+    /** The checksum of the local source file. This can be used to validate if
+     * the current on disk version matches the version described here.
+     *
+     * By default, this is not included in the output. Set the `checksums`
+     * option to `true` in the options to have this information populated. */
+    checksum?: string;
+    /** The path to an emitted version of the module, if the module requires
+     * transpilation to be loaded into the Deno runtime.
+     *
+     * By default, this is not included in the output. Set the `paths` option
+     * to `true` in the options to have this information populated. */
+    emit?: string;
+    /** The path to an optionally emitted source map between the original and
+     * emitted version of the file.
+     *
+     * By default, this is not included in the output. Set the `paths` option
+     * to `true` in the options to have this information populated. */
+    map?: string;
+    /** If when resolving the module, Deno encountered an error and the module
+     * is unavailable, the text of that error will be indicated here. */
+    error?: string;
+  }
+
+  /** An object representation of a Deno module dependency graph. */
+  interface ModuleGraph {
+    /** The root specifier for the graph. */
+    root: string;
+    /** The modules that are part of the graph. */
+    modules: ModuleGraphModule[];
+    /** The total size of all the unique dependencies in the graph in bytes. */
+    size: number;
+  }
+
+  interface InfoOptions {
+    /** If `true` populate the `checksum` property in the module graph
+     * entries. This is the checksum that is used by Deno to "lock" the
+     * dependency and could have security implications if exposed. Therefore,
+     * this is `false` by default. */
+    checksums?: boolean;
+    /** A string, URL pointing to a import map or an object providing the import
+     * map to be used when resolving modules.
+     *
+     * In order to resolve relative modules, import maps need a "base" which is
+     * the import map itself. When an object is passed as a value the current
+     * working directory for the Deno process will be used. */
+    importMap?: string | URL | ImportMap;
+    /** If `true` populate the `local`, `emit` and `map` fields in the module
+     * graph entries. These are fully qualified local filesystem paths, which
+     * could leak sensitive information. Therefore, this is `false` by default.
+     */
+    paths?: boolean;
+  }
+
+  /**
+   * **UNSTABLE**: new API, yet to be finalized.
+   *
+   * Similar to the command line functionality `deno info --json`, the API
+   * provides information about a module and its dependencies.
+   *
+   * Requires `allow/read` permissions to access local modules and cached
```

Comment:
Nitpick: change `allow/read` to `--allow-read` to keep consistent with other docs?